### PR TITLE
REBEL-620: Providing better alert information when developers are try…

### PR DESCRIPTION
**Pull request by sghiassy**
…ing to reach the dev menu when in build mode

### CODE REVIEW

File: React/Modules/RCTDevMenu.m [hunk 1]
```diff
@@ -514,6 +516,19 @@ - (NSString *)devMenuSettingsKey {
     return;
   }
 
+  BOOL inBuildMode = _bundleURL.fileURL == YES;
```
Comment (daniel-beard) [inline React/Modules/RCTDevMenu.m]: Don't need the `== YES`, just assign to the `.fileURL` value. 
http://stackoverflow.com/a/1934595/1144632

File: React/Modules/RCTDevMenu.m [hunk 2]
```diff
@@ -514,6 +516,19 @@ - (NSString *)devMenuSettingsKey {
...
+
+  if (inBuildMode) {
+    if (!alreadyShowedAlertWindow) {
+      alreadyShowedAlertWindow = YES;
+      [RCTDevMenu presentAlert:@"Debugging Disabled. See 'runMode' in Capacitorfile" onComplete:^{
+          alreadyShowedAlertWindow = NO;
```
Comment (daniel-beard) [inline React/Modules/RCTDevMenu.m]: What about `hasShownDisabledDebugAlert`

Comment (sghiassy) [inline React/Modules/RCTDevMenu.m]: +1

**Comment by sghiassy** [comment]
@daniel-beard - Updated

**Pull request merged**